apps/examples/client/index.coffee

```diff
@@ -18,10 +17,20 @@ module.exports = ->
       $(this).closest('.js-example').html singleExampleTemplate
         example: example
 
+      setupFollowButtons()
+
   # 
-  # Follow buttons
+  # Handle example channel click
   # 
+  $(document).on 'click', '.js-channel', (e) ->
+    e.preventDefault()
+    e.stopImmediatePropagation()
+    url = $(this).data('href')
+    window.open url
```

Comment:
Why?

Reply:
Don't open the lightbox when clicking on a block, just go to the channel

Reply:
hOW vErY FasciNaTIng